pkg/provider/azure_local_services.go

```diff
@@ -235,22 +234,18 @@ func (updater *loadBalancerBackendPoolUpdater) process(ctx context.Context) {
 // processError mark the operations as retriable if the error is retriable,
 // and fail all operations if the error is not retriable.
 func (updater *loadBalancerBackendPoolUpdater) processError(
-	rerr *retry.Error,
+	rerr error,
```

Comment:
why don't we need retriable error now?

Reply:
The status code is returned in azcore.ResponseError

Reply:
Will it retry automatically?

Reply:
https://github.com/kubernetes-sigs/cloud-provider-azure/blob/46be9622cf1d0760dede17bcd370ce89ea0f4255/pkg/azclient/policy/retryrepectthrottled/throttle.go#L42
if the response contains following status code, the request will be retried